features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2

```diff
@@ -434,8 +434,10 @@
       {% if apim.key_manager.enable_provisioned_app_validation is defined %}
       <EnableProvisionedAppValidation>{{apim.key_manager.enable_provisioned_app_validation}}</EnableProvisionedAppValidation>
       {% endif %}
-      {% if apim.key_manager.enable_apikey_subscription_validation is defined %}
-      <EnableAPIKeySubscriptionValidation>{{apim.key_manager.enable_apikey_subscription_validation}}</EnableAPIKeySubscriptionValidation>
+      {% if apim.key_manager.enable_lightweight_apikey_generation is defined %}
+      <EnableLightWeightAPIKeyGeneration>{{apim.key_manager.enable_lightweight_apikey_generation}}</EnableLightWeightAPIKeyGeneration>
+      {% else %}
+      <EnableLightWeightAPIKeyGeneration>true</EnableLightWeightAPIKeyGeneration>
```

Comment:
Shall we add this to the `default.json` without defining an else block?

Reply:
fixed